packages/frontend/src/common/components/navigation/Header.tsx

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
+'use client'
+import React, { useEffect, useState } from 'react'
 import Link from 'next/link'
 import Image from 'next/image'
 import { NavigationItem } from '@/common/components/navigation/NavigationItem'
+import { getUser, logOut } from '@/api'
+import { usePathname } from 'next/navigation'
+import { useRouter } from 'next/navigation'
+
+interface Props {
+  id?: string
+  message?: string
+}
 
 export function Header() {
+  const pathname = usePathname()
+  const router = useRouter()
+  const [user, setUser] = useState<Props>({})
```

Comment:
The name of the state is `user` but the type is `Props`. Could we improve that? :)